bootstrap/sql/mysql/v002__add_validation_level.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright 2017 Hortonworks.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+ALTER TABLE schema_metadata_info ADD validationLevel VARCHAR(255) NOT NULL AFTER compatibility;
```

Comment:
If its NOT NULL, should ensure previous rows have column populated with a default.